content/en/guides/get-started/routing.md

```diff
@@ -20,7 +20,13 @@ Learn more about [Routing](/guides/features/file-system-routing)
 
 ## Navigation
 
-To navigate between pages of your app, you should use the [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component) component. This component is included with Nuxt.js and therefore you don't have to import it like you do with other components. It is similar to the HTML `<a>` tag except that instead of using a `href="/about"` we use `to="/about"`. If you have used `vue-router` before, you can think of the `<NuxtLink>` as a replacement for `<RouterLink>`
+To navigate between pages of your app, you should use the [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component) component. This component is included with Nuxt.js and therefore you don't have to import it like you do with other components. It is similar to the HTML `<a>` tag, except that instead of using a `href="/about"` we use `to="/about"`. If you have used `vue-router` before, you can think of the `<NuxtLink>` as a replacement for `<RouterLink>`
+
+<base-alert type="info">
+
+Keep in mind, as you insert `<NuxtLink>` in your template, you may spell it out either as `<NuxtLink>` or `<nuxt-link>`
```

Comment:
Although you can use nuxt-link we are advising to use NuxtLink as a standard therefore do not want to add  nuxt-link here at least at beginner stage and not as a suggestion.

Reply:
Ah, I see - fixed - 4bd375a